lib/Form/Type/ContentType/FieldTypeChoiceType.php

```diff
@@ -64,7 +64,7 @@ public function getParent()
     private function getFieldTypeChoices()
     {
         $choices = [];
-        foreach ($this->fieldTypeCollectionFactory->getConcreteFieldTypesIdentifiers() as $fieldTypeIdentifier) {
+        foreach ($this->fieldTypeRegistry->getConcreteFieldTypesIdentifiers() as $fieldTypeIdentifier) {
```

Comment:
Repo-forms is kinda external package, so I'd rather use `\eZ\Publish\API\Repository\FieldTypeService` here.

Though it can be considered as out of scope, because we used FT Collection Factory before, which had the same issue.